start.go

```diff
@@ -17,25 +17,19 @@ func startCmd() cli.Command {
 		Usage:  "start a functions server",
 		Action: start,
 		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:  "log-level",
-				Usage: "--log-level DEBUG to enable debugging",
-			},
 			cli.BoolFlag{
 				Name:  "detach, d",
 				Usage: "Run container in background.",
 			},
+			cli.StringFlag{
+				Name: "config, c",
+				Usage: "Absolute path to Fn server configuration options file.",
```

Comment:
And why not call it `--env-file` like docker? Also, does it really require an absolute path?

Reply:
because we're not trying to reimplement docker CLI here, moreover, `--env-file` name is not verbose or helpful. The name of the opt should be very clear, that's why it was named as `--config`

Reply:
well, path doesn't suppose to be absolute, but it's more clear than even docker CLI helper string: `Read in a file of environment variables`

Reply:
But this isn't really a config file, it's a file containing env vars. If we ever support a config file, it probably (almost surely) wouldn't be in this format. This file format dates back to the Ruby days: https://github.com/bkeepers/dotenv

Also, the the flags should match docker run (like -d) since this is just a convenience command that passes right through to docker run.